portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,16 @@
   margin-left: auto;
```

Comment:
Can you put these in Alphabetical order? https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order